CONTRIBUTING.md

```diff
@@ -1 +1,64 @@
+# How to become a contributor and submit your own code
 
+## Contributor Agreements
+
+We'd love to accept your patches! Before we can take them, we have to jump a
+couple of hurdles.
+
+*   Do not use any unauthorised source codes,
+*   Feel free to PR even if the patch is very small,
+
+## Contributing A Patch
+
+1.  Submit an issue describing your proposed change to the
+    [issue tracker](https://github.com/CodewithSudeep/TodoGram/issues).
+2.  If you want to make a change that doesn't
+    have a corresponding issue in the issue tracker, please create one.
+3.  Also, coordinate with team members that are listed on the issue in question.
+    This ensures that work isn't being duplicated and communicating your plan
+    early also generally leads to better patches.
+4.  Fork the desired repo, develop and test your code changes. 
+5.  Ensure that your code adheres to the existing style in the sample to which
+    you are contributing.
+6.  Submit a pull request.
+
+## CodewithSudeep Community
+
+You are definitely encouraged to contribute to the discussion and you can also help us
+to keep the effectiveness of the group high by following and promoting the
+guidelines listed here.
+
+### Please Be Friendly
+
+Showing courtesy and respect to others is a vital part of the CodewithSudeep's culture,
+and we strongly encourage everyone participating in joining us in accepting nothing less. 
+Of course, being courteous is not the same as
+failing to constructively disagree with each other, but it does mean that we
+should be respectful of each other when enumerating the 42 technical reasons
+that a particular proposal may not be the best choice. There's never a reason to
+be antagonistic or dismissive toward anyone who is sincerely trying to
+contribute to a discussion.
+You don't have to actually submit code in order to sign up. Your participation
+itself is a valuable contribution.
+
+## Developing TodoGram
+
+This section discusses how to make your own changes to the Application.
+
+### Building TodoGram
+
+To make sure your changes work as intended and don't break existing
+functionality, you'll want to clone/fork and run. For that you can use:
+
+    git clone https://github.com/CodewithSudeep/TodoGram.git
+    cd TodoGram
+    npm install
+
+Make sure you have Node.js installed, as TodoGram is written
+in React.js.
+
+To build the app at last, do
+
+    npm run build
+
+Now the build should pass.
```

Comment:
Formatted Contributor documentation left for review.

Reply:
Looks good.